apartment_application_service/settings.py

```diff
@@ -236,6 +237,10 @@
 OIKOTIE_FTP_HOST = env("OIKOTIE_FTP_HOST")
 OIKOTIE_USER = env("OIKOTIE_USER")
 OIKOTIE_PASSWORD = env("OIKOTIE_PASSWORD")
+APARTMENT_DATA_TRANSFER_PATH = env("APARTMENT_DATA_TRANSFER_PATH")
+
+if not os.path.exists(APARTMENT_DATA_TRANSFER_PATH):
+    os.mkdir(APARTMENT_DATA_TRANSFER_PATH)
```

Comment:
I'm not sure if it's a good idea to create the directory when the settings are imported. But I don't have a good argument against it other than it somehow "feels" a bit dirty. 

Instead, maybe the functions that write the XML files in that directory should try to create it (or fail) if it does not exist? What do you think? 🙂